data/src/main/java/me/androidbox/data/remote/interceptor/TokenInterceptor.kt

```diff
@@ -0,0 +1,28 @@
+package me.androidbox.data.remote.interceptor
+
+import me.androidbox.domain.authentication.preference.PreferenceRepository
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class TokenInterceptor(private val preferenceRepository: PreferenceRepository) : Interceptor {
+
+    private companion object {
+        const val AUTHORIZATION_HEADER = "Authorization"
```

Comment:
This string will never change and this constant also doesn't add extra readability, so in that case I personally prefer to leave it